emissionsapi/config.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+'''
+Load and handle Emission API configuration.
+'''
+
+import yaml
```

Comment:
I think for yaml an external module is needed. Can you add it to the dependency list?